@xen-orchestra/lite/.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
+  "recommendations": ["Vue.volar"]
+}
```

Comment:
Is this something we wish to commit?

Reply:
It was committed on the `new-xo-lite` branch so far and I guess it can be useful for newcomers.

Reply:
Makes sense, thanks :slightly_smiling_face: